src/routes/photos.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import os
+import pickle
+
+from typing import List
+from fastapi import APIRouter, Depends, status, UploadFile, File, HTTPException, Form
+from sqlalchemy.orm import Session
+import cloudinary
+import cloudinary.uploader
+
+from src.database.db import get_db
+from src.repository import photos as repository_photos
+from src.schemas import PhotoResponse, PhotoModel, TagsPhoto
+from src.conf.config import settings
+
+router = APIRouter(prefix="/photos", tags=["photos"])
+
+
+@router.post('/', response_model=PhotoResponse)
+async def create_photo(file: UploadFile = File(),
+                       description: str = Form(),
+                       db: Session = Depends(get_db)):
+    cloudinary.config(
```

Comment:
Please initialize Cloudinary configuration globally one time before endpoints isntead of declare it in every endpoint

Reply:
ok. done